index.php

```diff
@@ -55,6 +55,14 @@
         unset($_GET['success']);
     }
 }
+
+//handle remove record request
+if (isset($_POST['remove'])) {
+    $selectedRecordID = $_POST['recordID'];
+    $recordModel->removeRecord($selectedRecordID);
```

Comment:
See my comment about making this method return a bool. You should add a basic check here to see if the delete worked or not